File: angular-food-delivery-website/src/app/user-dto.ts
export class UserDto {
  id: number;
  firstName: string;
  lastName: string;
  fullName: string;
  gender: Gender;
  phoneNumber: string;
  role: Role;
}

export enum Gender {
  Male = 1,
  Female = 2,
}

export enum Role {
  Admin = 1,
  Client = 2,
}

File: FoodDeliveryWebsite/FoodDeliveryWebsite.Models/Dtos/UserDtos/UserDto.cs
using FoodDeliveryWebsite.Models.Enums;

namespace FoodDeliveryWebsite.Models.Dtos.UserDtos
{
    public class UserDto
    {
        public int Id { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string FullName => $"{FirstName} {LastName}";

        public Gender Gender { get; set; }

        public string PhoneNumber { get; set; }

        public UserRole Role { get; set; }
    }
}
